### PR TITLE
feat: Integration with the rasa-bot chat

### PR DESCRIPTION
**Pull request by tczekajlo**
- Add integration with the rasa chart

### CODE REVIEW

**Comment by tczekajlo** [comment]
@melindaloubser1 @erohmensing The PR includes part related to the integration of the rasa bot chart with the rasa-x-helm chart, please 👀  if you have a minute and share thoughts 🙇🏻 🙏🏻

File: charts/rasa-x/values.yaml [hunk 1]
```diff
@@ -191,6 +191,19 @@ rasa:
...
+      enabled: true
+
+      # Define if external Rasa OSS should be used.
+      external:
+        # enable external Rasa OSS
+        enabled: true
+
+        # host of external Rasa OSS deployment
+        host: "http://rasa-bot"
+
```
Comment (melindaloubser1) [inline charts/rasa-x/values.yaml:206]:
```suggestion
      # You can disable the rasa-production deployment in order to use external Rasa OSS deployment instead.
      enabled: true

      # Define if external Rasa OSS should be used.
      external:
        # enable external Rasa OSS
        enabled: false

        # host of external Rasa OSS deployment
        host: "http://rasa-bot"

```
default should be false if enabled is default true

**Review by melindaloubser1 — APPROVED** [review]
This is clear to me 🚀

File: charts/rasa-x/values.yaml [hunk 2]
```diff
@@ -394,7 +408,7 @@ eventService:
 # app (custom action server) specific settings
 app:
   # default is to install action server from image.
-  install: true
+  install: false
```
Comment (erohmensing) [inline charts/rasa-x/values.yaml:411]: Why'd' this change?

File: README.md [hunk 3]
```diff
@@ -124,7 +124,41 @@ where `type` is the category of the change, `description` is a short sentence to
...
 For more information, please see [here](https://github.com/lob/generate-changelog#usage).
 
+## To 2.0.0
+
+The rasa-x-helm chart in version 2.0.0 supports using an external Rasa OSS deployment.
+
+### Enabling an external Rasa OSS deployment
+
+The rasa-x-helm chart >= 2.0.0 supports an option to use an external Rasa OSS deployment.
+Below you can find an example of configuration that uses the external deployment.
```
Comment (erohmensing) [inline README.md:134]: Should we make a note of how they need to configure either event broker to get it properly forwarding?

Comment (tczekajlo) [reply README.md:134]: Good point! I'll add it

File: charts/rasa-x/values.yaml (hunk 2 above)
Comment (tczekajlo) [reply charts/rasa-x/values.yaml:411]: Hmmm, I think after testing I forgot to change it back.

**Pull request merged**